h5p-bildetema/src/hooks/useL10n.ts

```diff
@@ -18,3 +18,14 @@ export const useL10ns = <TString extends TranslationKey = TranslationKey>(
   });
   return translations;
 };
+
+export const useTranslation = (): ReturnType<
+  typeof useTranslationOriginal<TranslationKey>
+> => {
+  const original = useTranslationOriginal();
+
+  return {
+    ...original,
+    t: (key: TranslationKey) => original.t(key),
+  };
+};
```

Comment:
This hook was created to allow the use of the `t` function within components, instead of carefully fetching all translations at the start of the component (even ones which aren't in use).

I'll rollback this change if you disagree.

Reply:
The hook is good to use, but I think it could be added in a different PR perhaps. So, that if we want to use it we can change other imports of translations elsewhere at the same time. 

We do have some translations and a11y improvements (as you also nicely suggested) that we are going to apply now that the new functionally is in place.